test/configs/webpack-debugging.config.js

```diff
@@ -0,0 +1,89 @@
+const devCerts = require("office-addin-dev-certs");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const CustomFunctionsMetadataPlugin = require("custom-functions-metadata-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require('path');
+const webpack = require("webpack");
+
+module.exports = async (env, options) => {
```

Comment:
Why a separate config for debugging? It's hard to see what is different for this file.

Reply:
The only difference is on line 84: the port which dev server runs.
If I don't change ports, the second test to get executed sometimes detects that the first dev-server is still running even if it is not.
This happens because our dev-server takes some time to close properly.

Reply:
Perhaps you could provide a way to override this when doing the debugging tests but not require the separate config file.

Reply:
I will try to come up with a way to pass the debugging dev-server port without having to write a whole new file.

Reply:
@akrantz I couldn't figure how to change the port easily through any existing API on our end. As I would have to change both manifest and webpack-config, I don't know how I can do this without having to read the file and edit it brute force.